.circleci/verbatim-sources/nightly-binary-build-defaults.yml

```diff
@@ -51,18 +52,22 @@ binary_populate_env: &binary_populate_env
     fi
 
     # We put this here so that OVERRIDE_PACKAGE_VERSION below can read from it
-    export PYTORCH_BUILD_VERSION="1.0.0.dev$(date +%Y%m%d)"
+    export DATE="$(date -u +%Y%m%d)"
+    export PYTORCH_BUILD_VERSION="1.0.0.dev$DATE"
     export PYTORCH_BUILD_NUMBER=1
 
     cat >>"$envfile" <<EOL
     # =================== The following code will be executed inside Docker container ===================
+    export TZ=UTC
```

Comment:
Does `TZ` need to be exported again here, or would it inherit the earlier export on line 18?

Reply:
It definitely won't inherit during the binary test jobs when start a new docker and want the env in there. I think I remember it sometimes not inheriting on mac jobs or other jobs? I don't really remember. I chose to just always source the env again in every step rather than figure out when it wouldn't inherit